apiserver/common/storagecommon/storage.go

```diff
@@ -57,6 +58,10 @@ type StorageInterface interface {
 	// BlockDevices returns information about block devices published
 	// for the specified machine.
 	BlockDevices(names.MachineTag) ([]state.BlockDeviceInfo, error)
+
+	// UnitStorageAttachments returns the storage attachments for the
+	// identified unit.
```

Comment:
sure, though several of the methods above use identified as well :)  I've changed them all